router/batchrouter/asyncdestinationmanager/snowpipestreaming/snowpipestreaming.go

```diff
@@ -31,20 +32,21 @@ import (
 	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/snowpipestreaming/internal/model"
 	"github.com/rudderlabs/rudder-server/utils/misc"
 	"github.com/rudderlabs/rudder-server/utils/timeutil"
+	"github.com/rudderlabs/rudder-server/warehouse/integrations/manager"
 	whutils "github.com/rudderlabs/rudder-server/warehouse/utils"
 )
 
 var json = jsoniter.ConfigCompatibleWithStandardLibrary
 
 func New(
 	conf *config.Config,
-	logger logger.Logger,
+	mLogger logger.Logger,
```

Comment:
`mLogger` Do you mean module logger? Can we rename it as log?

Reply:
I had renamed it to `mLogger` since `managerCreator` had an argument of type `logger.Logger`. Renaming it to `log`